nncf/common/statistics.py

```diff
@@ -53,6 +54,16 @@ def rb_sparsity(self) -> Optional[RBSparsityStatistics]:
         """
         return self._storage.get('rb_sparsity')
 
+    @property
+    def movement_sparsity(self) -> Optional[MovementSparsityStatistics]:
```

Comment:
create unit test

Reply:
added in https://github.com/vuiseng9/nncf/pull/10/commits/9e1f2428016be40ccd1129f59038dd6d399db0fe